linux_os/guide/system/auditing/policy_rules/audit_delete_success/rule.yml

```diff
@@ -55,3 +55,15 @@ template:
         filepath: /etc/audit/rules.d/30-ospp-v42-4-delete-success.rules
         contents: |+
             {{{ file_contents_audit_delete_success|indent(12) }}}
+
```

Comment:
this file is not relevant anymore for this SRG, it was updated recently... please rebase on top of master